### PR TITLE
feat(password-validator): add types

### PR DESCRIPTION
**Pull request by MadDeveloper**
Using flow v0.134 and password-validator v4.1.1.

- Links to documentation: https://github.com/tarunbatra/password-validator/wiki
- Link to GitHub or NPM: https://github.com/tarunbatra/password-validator
- Type of contribution: new definition

### CODE REVIEW

File: definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/password-validator_v4.x.x.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// flow-typed signature: 3587dd2cb279eaf68a55c988c6bcb495
+// flow-typed version: <<STUB>>/password-validator_v4.x.x/flow_v0.134.0
```
Comment (pascalduez) [inline definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/password-validator_v4.x.x.js:2]: Please remove those comments, they are added by the CLI.

File: definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/test_password-validator_v4.x.x.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      .has().not().spaces()
+      .has(/(.*[A-Z]){2,}/);
+
+    schema.validate('3L}Ls4[4}qQB>UgN');
+  });
+
+  it('should fail with incorrect arguments', () => {
+    const schema = new PasswordValidator();
+
+    // $FlowExpectedError
```
Comment (pascalduez) [inline definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/test_password-validator_v4.x.x.js:27]: You're missing error codes, check the failing tests.
E.g.: `$FlowExpectedError[extra-arg]`

**Review by pascalduez — CHANGES_REQUESTED** [review]
Thanks!

A few modifications and we are good to go.

File: definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/password-validator_v4.x.x.js (hunk 1 above)
Comment (MadDeveloper) [reply definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/password-validator_v4.x.x.js]: Updated.

File: definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/test_password-validator_v4.x.x.js (hunk 2 above)
Comment (MadDeveloper) [reply definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/test_password-validator_v4.x.x.js]: Updated.

**Review by pascalduez — APPROVED** [review]

**Pull request merged**